aip/0132.md

```diff
@@ -127,8 +128,8 @@ request message **should** contain a `string order_by` field.
   `"foo,bar"`.
 - The default sorting order is ascending. To specify descending order for a
   field, users append a `" desc"` suffix; for example: `"foo desc, bar"`.
-- Redundant space characters in the syntax are insignificant. `"foo, bar desc"`
-  and `" foo , bar desc "` are equivalent.
+- Redundant space characters in the syntax are insignificant. `"foo, bar desc"`,
```

Comment:
Again, can't comment directly where I wanted to. Is there any guidance about deterministic results when there is no 'order_by'? In addition, if there is an 'order_by', but it's unset, is there guidance about default sort order?

Reply:
My gut feeling is that it should be deterministic, and the service may decide what to do beyond that. Thoughts?

Reply:
Sounds good. Although deterministic may not be guaranteed due to consistency issues. However, as an API consumer, it would be nice to know if I call the same List method repeatedly with exactly the same parameters, that the response should be the same (or as much as it can taking into consideration consistency).